README.md

```diff
@@ -1,77 +1,10 @@
-  
-  Furthermore, to avoid a current issue ([#16](/../../issues/16)), you need to do the following as a workaround:
-  
-  * go to the directory you specified in the last line of the config file. 
-  * open `node_modules/d3/build/d3.node.js` (same on every platform).
-  * comment out every line containing `d3-request`. 
-    
-
-* Get data for the example app. For MySQL, download a small nba database from (https://www.dropbox.com/s/3chn6r73vzxttr2/nba_db_mysql.sql?dl=0). Load it into your MySQL as follows:
-
-      $ mysql -u username -p                     # login to mysql
-      $ Enter password:                          # enter password if necessary
-      $ > create database nba                    # create a database 'nba'
-      $ > exit                                   # log out
-      $ mysql -u root -p nba < nba_db_mysql.sql  # import 
-   
-   For PostgreSQL, download the nba database from (https://www.dropbox.com/s/baqb01thxvfthk5/nba_db_psql.sql?dl=0). Load it into your PostgreSQL as follows:
-   
-      $ psql postgres               # login as postgres
-      $ Enter password:             # enter password if necessary
-      $ > create database nba       # create a database 'nba'
-      $ > \q                        # log out
-      $ psql nba < nba_db_psql.sql  # import    
-
-* Go to `tile-server/`. Run `mvn compile` to build the server. Run `mvn exec:java -Dexec.mainClass="main.Main"` to start the server. After the server starts, it will prompt that it did not find the spec of the app and is waiting for it. 
-
-* Write spec according to the spec language documentation (to be added). The spec for the example app is in `compiler/examples/nba/nba.js`. 
-
-* Run the spec using node.js. To run the example spec, run
-
-      $ cd compiler/examples/nba
-      $ node nba.js
-    
-    Three things to expect after running the spec: 
-    (1) a `Kyrix` database will be created (if not existed), (2) a `project` table will be created (if not existed) to store the specs of the apps and (3) the compiler will notify the tile server that a new spec is ready. The tile server will then start building some indexes. 
-
-* Open your browser and go to `http://localhost:port#` to see the initial canvas. 
-
-* You can debug by modifying the spec and running it again. The tile server will be notified of the changes every time you run a spec, and will recalculate the indexes accordingly. 
+The easiest way to run Kyrix is to use our docker-compose config. Simply install docker and docker-compose (v3 or later, i.e. don't trust pre-installed versions), then from the main directory run `docker-compose up` which will build and startup Kyrix using the supplied docker-compose.yml file, including a Postgres instance. Allow 10+ minutes for fresh installs to startup, depending on your network and CPU. Google Cloud users: we recommend at least 4 vCPU and 6 GB of RAM.
```

Comment:
Several notes: 
1. Some tips on how to install docker would be better for docker newbies... for instance, I used [this workaround](https://unix.stackexchange.com/questions/363048/unable-to-locate-package-docker-ce-on-a-64bit-ubuntu) to install docker-ce on ubuntu 16.04. But this is not needed for now, and I can add later.
2. only mentioning google cloud seems a little weird tbh... shouldn't that spec also roughly work for aws/azure?
3. It would be better to include a doc on the typical development workflow of using the docker image...

Reply:
rewritten.